config/201-controller-role.yaml

```
@@ -73,7 +73,7 @@ rules:
     verbs: ["create", "get", "list", "update"]
   - apiGroups: ["tekton.dev"]
     resources: ["pipelineruns"]
-    verbs: ["get", "delete", "list", "create", "watch", "update"]
+    verbs: ["get", "delete", "list", "create", "watch", "update", "patch"]
```

If I'm not wrong `status.go` was the only one doing an `update` request. If so, now with `patch` we don't need an `update` verb?
In case, the above is a wrong statement - we could instead do a
```suggestion
    verbs: ["*"]
```
since we're already using all the Verbs?

---

yeah Thanks! will do the rbac cleanup in a separate patch.

regarding using `*` always better to write verbs explicitly 🙃 
also customers don't like `*` 😄

---

ah, noted. Thanks